addons/account/migrations/13.0.1.1/end-migration.py

```diff
@@ -44,6 +44,15 @@ def add_default_account_data(env):
             company[company_field] = account.id
 
 
+def update_mail_alias_for_moves(env):
+    # Done here just in case alias customizations exist
+    journals = env["account.journal"].search([("alias_id", "!=", False)])
```

Comment:
```suggestion
    journals = env["account.journal"].with_context(active_test=False).search([("alias_id", "!=", False)])
```
?

Reply:
Yes, I have just checked that disable journals still preserve the alias.